contracts/.solhint.other.json

```diff
@@ -33,7 +33,7 @@
     "constructor-syntax": "error",
     "gas-calldata-parameters": "warn",
     "gas-custom-errors": "warn",
-    "gas-increment-by-one": "warn",
+    "gas-increment-by-one": "off",
```

Comment:
This solhint config only effects tests and scripts, not the production code.
In these cases, we don't care about gas optimizations.